style.css

```diff
@@ -0,0 +1,70 @@
+.row {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    width: 100%;
+}
+
+.column {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex: 1;
+}
+
+.double-column {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex: 2;
+}
+
+.triple-column {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex: 3;
+}
+
+a {
+    text-decoration: none;
+    text-align: center;
+    color: black;
+}
+
+.grid-container {
+    display: grid;  
+    grid-template-areas: 'header'
+    'main'
+    'footer';
+    grid-gap: 10px;
+    padding: 10px;
+    text-align: center;
+    grid-template-rows: 50px 500px 50px;
+}
+
+#header-placeholder {
+    grid-area: header;
+}
+
+#footer-placeholder {
+    grid-area: footer;
+}
+
+#main-placeholder {
+    grid-area: main;
+}
+
+#invisible-forest {
+    color: white;
+}
+
+iframe {
+    height: 100%;
+    border: none;
+    -ms-transform:scale(0.90);
+    -moz-transform:scale(0.90);
+    -o-transform: scale(0.90);
+    -webkit-transform: scale(0.90);
+    transform:scale(0.90);
+}
```

Comment:
The way this scaling ends up means that the content in the `iframe` usually requires a scroll bar, even though there's room on the parent screen to show it all. Playing with the aspect ratio of the `iframe` to better match the content can help resolve some of this.